gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,7 +1,9 @@
-#Fri Aug 19 10:56:09 IST 2022
+#Thu May 30 09:49:42 IST 2024
 distributionBase=GRADLE_USER_HOME
-distributionUrl=https\://services.gradle.org/distributions/gradle-8.2.1-bin.zip
 distributionPath=wrapper/dists
-zipStorePath=wrapper/dists
+distributionSha256Sum=544c35d6bd849ae8a5ed0bcea39ba677dc40f49df7d1835561582da2009b961d
```

Comment:
Since you updated the wrapper, maybe we should also add [a github actions for validation](https://github.com/gradle/wrapper-validation-action).

Reply:
Done